### DISCUSSION TITLE
Where is the profile image gone?

### DISCUSSION THREAD

**Discussion by buzter** [kimai/kimai#2783] [Questions & Answers] [answered]
I updated to 1.15 resp. 1.15.1. I can't find the input field for the profile images anymore. What did I miss?

![grafik](https://user-images.githubusercontent.com/19949143/134015681-4e3f5fbe-74ad-4e3b-9537-2775d9a4f523.png)

**Comment by kevinpapst** [comment, marked as answer]
There is a new system-setting under `Theme`  to activate the field. 
Adding URLs is a potential security risk (at least privacy issue) so it was removed by default.

You can activate it:
<img width="298" alt="Bildschirmfoto 2021-09-20 um 16 04 48" src="https://user-images.githubusercontent.com/533162/134016093-50e8e12c-5109-4417-9e1e-75ff26cab55b.png">

**Comment by buzter** [comment]
Thanks  and sorry for net getting that. crazy, i think I looked through the settings almost 10 times - and still overlooked this point. PEBKAC

**Reply by kevinpapst** [reply]
now we have a posting to link to in the future 😁
